packages/create-workleap/index.ts

```diff
@@ -0,0 +1,109 @@
+import * as child_process from "child_process";
```

Comment:
You can deconstruct directly in the import statement

`import { exec } from "child_process"`;